src/runtime-rs/crates/runtimes/common/src/sandbox.rs

```diff
@@ -12,6 +12,7 @@ pub trait Sandbox: Send + Sync {
     async fn start(
         &self,
         netns: Option<String>,
+        network_created: bool,
```

Comment:
Can we use some check function instead of passing this parameter?

Reply:
As we need to determine if the `netns` is created by kata or contained, the best way is to do this is to check the existence of `netns` in OCI spec. If `netns` is None, kata will create it and set `network_created` to true, otherwise, to None, so I think no more check functions are needed.